index.html

```diff
@@ -64,6 +64,36 @@
         </gu-column>
       </gu-row>
 
+      <h2>Boxes</h2>
+      <gu-row>
+        <gu-column span="4">
+          <gu-box variant="primary">
+            This is some content
+          </gu-box>
+        </gu-column>
+        <gu-column span="4" gu-box="secondary">
```

Comment:
Ok in that case I agree an attribute makes sense. The word "box" is a bit ambigous. It could be something you put things in. Maybe another name that would be better? `gu-area`, `gu-field` ... `gu-<something>`?